edx_lint/pylint/layered_test_check.py

```diff
@@ -32,32 +52,13 @@ class LayeredTestClassChecker(BaseChecker):
     }
 
     @utils.check_messages(MESSAGE_ID)
```

Comment:
No, in fact, that was one of the first things I changed.  That decorator is used on visit_ methods to limit the exploration. Not sure why we had it on a helper method originally.